apps/api/test/functional/create-wallet.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { faker } from "@faker-js/faker";
+import { eq } from "drizzle-orm";
+import { container } from "tsyringe";
+
+import { app, initDb } from "@src/app";
+import { USER_WALLET_SCHEMA, UserWalletSchema } from "@src/billing/providers";
+import { ApiPgDatabase, POSTGRES_DB } from "@src/core";
+import { closeConnections } from "@src/db/dbConnection";
+
+jest.setTimeout(30000);
```

Comment:
was this for testing?

Reply:
Request was taking more then default 5sec for me quite a bit